### PR TITLE
Add open endpoint for scheduler callbacks

### PR DESCRIPTION
**Pull request by imsickofmaps**
Not using DRF, just open `/subscriptions/id/send` expecting sequence number in the body, triggers send message celery task

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@gsvr ready for review

File: mama_ng_control/apps/subscriptions/views.py [hunk 1]
```diff
@@ -14,3 +20,47 @@ class SubscriptionViewSet(viewsets.ModelViewSet):
     serializer_class = SubscriptionSerializer
     filter_fields = ('contact', 'messageset_id', 'lang', 'active', 'completed',
                      'schedule', 'process_status', 'metadata',)
+
+
+class SubscriptionSend(APIView):
+
+    """
+    Triggers a send for the
```
Comment (gsvr) [inline mama_ng_control/apps/subscriptions/views.py]: Incomplete sentence.

File: mama_ng_control/apps/subscriptions/views.py [hunk 2]
```diff
@@ -14,3 +20,47 @@ class SubscriptionViewSet(viewsets.ModelViewSet):
...
+        """
+        # Look up subscriber
+        subscription_id = kwargs["subscription_id"]
+        try:
+            subscription = Subscription.objects.get(id=subscription_id)
+            expect = ["message-id", "send-counter", "schedule-id"]
+            if set(expect).issubset(request.data.keys()):
+                # Set the next sequence number
+                subscription.next_sequence_number = request.data[
+                    "send-counter"]
```
Comment (gsvr) [inline mama_ng_control/apps/subscriptions/views.py:44]: Without looking further into this, should this have a +1?

Comment (gsvr) [inline mama_ng_control/apps/subscriptions/views.py:44]: Should there not be some kind of check for the max_sequence_number?

Comment (imsickofmaps) [inline mama_ng_control/apps/subscriptions/views.py:44]: No, those things will be handled by the scheduler and the sending tasks

**Pull request merged**